spec/openapi.yaml

```diff
@@ -83,6 +118,43 @@ components:
         - text
         - firmware
 
+    Storage:
+      type: object
+      properties:
+        name:
+          type: string
+          example: 'PrusaLink gcodes'
+        type:
+          type: string
+          description: Storage source
+          enum: ["LOCAL", "SDCARD", "USB"]
+        path:
+          type: string
+          description: Path to storage (not display path)
+          example: "/local"
+        print_files:
+          type: integer
+          description: Size of all print files in bytes
+          example: 19216842
+        system_files:
+          type: integer
+          description: Size of all system files in bytes
+          example: 4242
+        free_space:
+          type: integer
+          description: System free space in bytes
+          example: 1921681142
+        total_space:
+          type: integer
+          description: System total space in bytes
+          example: 8589934592
+        available:
+          type: boolean
+          description: Whether the storage is available or not
+          example: True
+      required:
+        - available
```

Comment:
missing:
* path - that we always know
* type - at this moment, we still always know, when was changed (SDCARD->SDCARD_WIFI), don't care about that